.github/workflows/ci.yml

```diff
@@ -639,7 +639,7 @@ jobs:
         run: make docker-verify
       - name: Test Reporter
         if: always()
-        uses: phoenix-actions/test-reporting@v12
```

Comment:
Please update this aswell https://github.com/apache/nifi-minifi-cpp/blob/d3d205f3c8a8376bbad4c8a47758d585d57c653b/.github/workflows/verify-python-compatibility.yml#L86

Reply:
thanks, done